sdk/appconfiguration/app-configuration/src/models.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { RequestOptionsBase, HttpResponse } from "@azure/core-http";
+import { OperationOptions, HttpResponse } from "@azure/core-http";
```

Comment:
Can you file an issue on `autorest.typescript` (if we haven't already) that tracks the code generator changes necessary to make this get generated in the future?